nixos/modules/programs/java.nix

```diff
@@ -38,11 +39,7 @@ in {
         type = types.package;
       };
 
-      binfmt = mkOption {
-        type = types.bool;
-        default = false;
-        description = lib.mdDoc "Register Java program execution with binfmt";
-      };
+      binfmt = mkEnableOption (lib.mdDoc "Register Java program execution with binfmt");
```

Comment:
This will make an option with the description "Whether to enable Register Java program execution with binfmt", which is a bit awkward

What about something like "Java program execution with binfmt" or "binfmt registration for Java program execution"? (so that way it would come out to "Whether to enable Java program execution with binfmt" or "Whether to enable binfmt registration for Java program execution")

Alternatively it can go back to the original option you had here (as I mentioned it's more of a preference thing, and I was mostly just letting you know it existed for constructs like that)

Reply:
Fixed

Reply:
Done